assets/shaders/RayQuery/RayQuery.hlsl

```diff
@@ -0,0 +1,102 @@
+// https://nvpro-samples.github.io/vk_mini_path_tracer/index.html
+// https://github.com/nvpro-samples/vk_mini_path_tracer/blob/main/vk_mini_path_tracer/shaders/raytrace.comp.glsl
+
+// Limit the kernel to trace at most 64 samples.
+#define NUM_SAMPLES 64;
+
+// Limit the kernel to trace at most 32 segments.
+#define NUM_TRACED_SEGMENTS = 32;
+
+RWTexture2D<float4> imageData;
+RaytracingAccelerationStructure tlas;
+
+[numthreads(8, 8, 1)]
+void MainCS(uint3 Gid  : SV_GroupID,          // current group index (dispatched by c++)
+            uint3 DTid : SV_DispatchThreadID, // "global" thread id
+            uint3 GTid : SV_GroupThreadID,    // current threadId in group / "local" threadId
+            uint GI    : SV_GroupIndex)       // "flattened" index of a thread within a group)
+{   
+    // The resolution of the buffer, which in this case is a hardcoded vector
+    // of 2 unsigned integers:
+    const uint2 resolution = uint2(512, 512);
```

Comment:
It would be great to use the application window size.

Reply:
It does not block the pull request.

Reply:
Done!